src/domogik/common/queryconfig.py

```diff
@@ -143,4 +150,105 @@ def query(self, type, name, key = None):
                 self._log.error("Query config : error returned. Reason : {0}".format(dat['reason']))
                 return None
 
+class QueryDB():
```

Comment:
Yeah, I had discussion with fritz about it.  It is indeed a matter of principle about plugin db access.
It's why I preserve DbHelperMQ by optional choice.
For moment we decide to give preference to optimizing the speed rather than idea plugin never access on db. (see fritz for more details)

Concerning plugin start, there some start errors by using start from admin (see #480) directly affected by this patch. And it improve significatively start from manager (TestPlugin and domogik start)